### PR TITLE
feat: added permission directive

### PR DESCRIPTION
**Pull request by chenshoo**

### CODE REVIEW

File: packages/polaris-schema/src/utils/executable-schema-creator.ts [hunk 1]
```diff
@@ -18,6 +19,9 @@ export function makeExecutablePolarisSchema(
 ): GraphQLSchema {
     const mergedTypes = getMergedPolarisTypes(typeDefs);
     const mergedResolvers = getMergedPolarisResolvers(resolvers);
+    schemaDirectives
+        ? (schemaDirectives.permissions = PermissionsDirective)
+        : (schemaDirectives = { permission: PermissionsDirective });
```
Comment (ronkatz96) [inline packages/polaris-schema/src/utils/executable-schema-creator.ts]: schemaDirectives
        ? (schemaDirectives.permissions = PermissionsDirective)
        : (schemaDirectives = { permissions: PermissionsDirective });

File: packages/polaris-schema/src/directives/permissions-directive.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            actions,
+        );
+
+        context.permissionsContext.digitalFilters = result.digitalFilters || {};
+        context.permissionsContext.portalData = result.portalData || {};
+        if (!result.isPermitted) {
+            throw new Error('Forbidden');
+        }
+    } else if (context.requestHeaders.requestingSystemId) {
+        // implement class
```
Comment (ronkatz96) [inline packages/polaris-schema/src/directives/permissions-directive.ts]: to do

File: packages/polaris-core/src/config/permissions-configuration.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { PolarisGraphQLContext } from '@enigmatis/polaris-common';
+
+export interface PermissionsConfiguration {
+    customPermissionsFunction?: (
```
Comment (chenshoo) [inline packages/polaris-core/src/config/permissions-configuration.ts]: not sure about the name, cause its only activated if you has requesting system.
maybe systemPermissionsFunction?

File: packages/polaris-schema/src/directives/permissions-directive.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+): Promise<void> {
+    if (context.requestHeaders.upn) {
+        context.permissionsContext = {
+            ...context.permissionsContext,
+            digitalFilters: {},
+            permissionsCacheHolder: new PermissionsCacheHolder(),
+            portalData: undefined,
+        };
+
+        const wrapper = new PermissionsServiceWrapper(
```
Comment (ronkatz96) [inline packages/polaris-schema/src/directives/permissions-directive.ts]: permissionsServiceWrapper

**Review by ronkatz96 — APPROVED** [review]

**Pull request merged**